packages/editor/src/store/selectors.js

```diff
@@ -195,7 +195,7 @@ export function getCurrentPostType( state ) {
  *
  * @param {Object} state Global application state.
  *
- * @return {?number} ID of current post.
+ * @return {?(string|number)} Template slug or current post ID.
```

Comment:
```suggestion
 * @return {?(string|number)} The current post ID or template slug.
```

Let's emphasize the post ID.

Reply:
Good point. Updated in https://github.com/WordPress/gutenberg/pull/71347/commits/7d658bbe4a0360ebb399a1753c3a93b2280b1926. I swapped types as well so there's more logical order.

Reply:
Thanks, @kmanijak!